### DISCUSSION TITLE
Per-repository reviewer/assignee in main config

### DISCUSSION THREAD

**Discussion by FabianoAlm** [renovatebot/renovate#30006] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Gitlab

### Please tell us more about your question or problem

Maybe I have missed this somewhere in the documentation, but is there a way to set renovate to use specific reviewers/assignees for specific groups/repos (without using CodeOwners)? I am currently running Renovate on a Gitlab instance with a rather large number of projects and I do not wish to introduce a renovate file to each one. I would like, however, to set up Renovate to use specific reviewers for the merge requests for each group/repo. I would also be fine with a solution that, at runtime, introduces the renovate file after cloning and then creates the merge request without it.

**Comment by RahulGautamSingh** [comment, marked as answer]
Combining [matchRepositories](https://docs.renovatebot.com/configuration-options/#matchrepositories) with [reviewers](https://docs.renovatebot.com/configuration-options/#reviewers) should do the job.
